OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -153,12 +153,19 @@ List<CPos> CalculatePathToTarget(Actor self)
 				return pathFinder.FindBidiPath(fromSrc, fromDest);
 		}
 
-		public override IEnumerable<Target> GetTargets(Actor self)
+		public override Color TargetLineColor
 		{
-			if (inner != null)
-				return inner.GetTargets(self);
+			get
+			{
+				if (NextActivity != null)
+					return NextActivity.TargetLineColor;
+				return Color.Green;
+			}
+		}
 
-			return Target.None;
+		public override IEnumerable<Target> GetTargets(Actor self)
+		{
+			yield return Target;
```

Comment:
It was removed for simplification. What ever it does in the inner loop, it is moving towards target and the next activity is more important. Landing sequence for example, I wanted to simplify it and prevent moving to the left of the airfield before Land activity to be not drawn but I couldn't do that neatly.